data/wanderer/wanderers start.txt

```diff
@@ -84,6 +84,61 @@ event "label wanderer space"
 		sprite "label/wanderers"
 
 
+mission "Wanderers: Unfettered Diplomat"
+	landing
+	source
+		attributes "unfettered"
+	to offer
+		has "First Contact: Wanderer: offered"
+		has "First Contact: Unfettered: offered"
+		not "unfettered know of wanderers"
+		not "Wanderers: Hai Diplomat: offered"
+		not "Wanderers: Diplomacy: offered"
+	on offer
+		conversation
+			`The aliens north of Hai space seem to speak the Hai language. Do you want to look for someone here who could serve as an interpreter?`
+			branch hai
+				has "First Contact: Hai"
+			choice
+				`	(Yes.)`
+					goto try
+				`	(Not right now.)`
+					defer
+			label hai
+			choice
+				`	(Yes.)`
+				`	(No; the Fettered Hai are better suited for this.)`
```

Comment:
```suggestion
				`	(No; the Fettered Hai are better suited for this.)`
```

I'd suggest reworking this bit. Something like "the peaceful Hai" or the "Hai central government."

"Fettered" is not the official name, and if choosing this option, player is probably not fully sharing in the Unfettered bias of using that phrase.

Reply:
Changed to "the Hai southern of here"